### DISCUSSION TITLE
Trigger workflows on event payload

### DISCUSSION THREAD

**Discussion by jasonkuhrt** [community/community#11745] [Actions]
Currently it is not possible to trigger a workflow on a payload.

This is pretty annoying for us because of the noise created in PRs. Take a looks:

<img width="882" alt="CleanShot 2022-02-18 at 18 39 59@2x" src="https://user-images.githubusercontent.com/284476/154775123-47882155-6261-4723-8b6c-338e4b0a4167.png">

There are a bunch of skipped checks because they are for deployment_status successful, not pending.

It would be very nice to push this condition higher in the workflow to the point where the entire thing only runs given it passes the payload shape we want.

Thanks!

**Comment by ethomson** [comment]
Thanks for the feedback @jasonkuhrt - I agree.  I'd like to put some more ability to add conditionals into the triggers - we'll put this on the backlog but I'm afraid that I don't have an ETA yet.
